### DISCUSSION TITLE
4주차 미팅

### DISCUSSION THREAD

**Discussion by sejoonkimmm** [GunGonGamLee/ft_transcendence#90] [General]
### 해야할 일
* 각 기능들 API 정하기
* Hashicorp vault https로 접근 가능하게 수정
* DEBUG모드 False일 때 Hashicorp Vault에서 제대로 비밀들 참조할 수 있도록
* Login 페이지 가상화에서 테스트 완료 / 다양한 경우 테스트까지 완료하는 스크립트 test.py에 작성하기

### 논의할 것
* 차차 수정

**Comment by nyj001012** [comment]
# 공통
## 설날에는 올 것인가?
- 올 수 있는 사람은 오고 가족끼리 보낼 사람은 보내고
- sejokim: 풀출석
- wonlim: 14일 까지 부재
- hyojocho: 풀출석
- donghyk2: 풀출석
- yena: 풀출석

# 진행상황 공유
## 백엔드
- 이번 주 내로 로그인 로직 구현 완료

## 프론트엔드
- 로그인 페이지 구현 완료
- 인증, 닉네임 생성 페이지 구현 완료
- 사용자 전적 페이지 구현중

**Comment by bluedog129** [comment]
## 프론트엔드
- 로그인페이지 구현
- 사용자 지정 게임 

## 기획
- 친구창 모달
- 전체랭킹

**Comment by LWJ0513** [comment]
1월 24일 안에 로그인 끝내기
api 명세 최대한 끝내서 api 짜는게 목표입니다~
